### PR TITLE
Create Home and Details Pages

### PR DESCRIPTION
**Pull request by kwekubright**
### Description:
In this feature, I achieved the following:

-  Creates a Home page ✅ 
- When the page loads, the web app shows the list of categories that could be filtered out by asset name. ✅ 
- Created Details page to retrieve data from the API to show detailed data of the selected asset. ✅ 
- I configured redux ✅ 
- Wrote some test as well ✅ 

### The result
cryptobanner

### CODE REVIEW

File: README.old.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Covid Tracker App
+
```
Comment (danmainah) [inline README.old.md:2]: - [ ] Kindly remove this old readme file, since you have another one already created.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,65 @@
...
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Prerequisites
+
+- Node
+- Git
+- Code Editor
+
+## Insall
```
Comment (danmainah) [inline README.md]: - [ ] Kindly check the spelling of your title. it's currently misspelled.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,65 @@
...
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
```
Comment (danmainah) [inline README.md:63]: - [ ] Kindly give credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) in your acknowledgments section.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @kwekubright , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️

- [ ] @kwekubright great work adding tests, kindly add more tests for your app and ensure you have tests for some redux functions. Also, don't forget to include integration tests in your tests.
` hint`
You could add a test that performs a mocking data fetching and Redux Store connection.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by ShadyShawkat — APPROVED** [review]
## Hi @kwekubright, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**